styles/controllers/grid/files/icons.less

```diff
@@ -14,36 +14,36 @@
 	/* -- Iconography: http://www.jankoatwarpspeed.com/post/2009/09/17/sixpack-status-icons.aspx --*/
 
 	.pkp_controllers_grid_files_response pkp_controllers_grid_files_success {
-		background: #daf0da url({$baseUrl}/lib/pkp/templates/images/structure/response_success.png);
```

Comment:
Ok I've gone to `@baseUrl` for this PR.

Would you like to switch all the LESS variables over to camelcase? I'm fine with that, but I think I'd do it in a separate PR as it will add a lot of noise to this one (it will just be string replacements across every LESS file in ojs/omp/pkp-lib).

If so let me know and I'll create an issue so I remember to do this against a clean master branch before OJS 3.